hacking/target-cortex-a.md

```diff
@@ -5,11 +5,11 @@ This captures some discussion from Gitter on Cortex-A driver implementation.
 The Cortex-A driver supports ARMv7-A with a little Zynq-7000 specific code. ARMv8 has some in common, but many significant differences.
 
 These Cortex-A variants should already be detected correctly: A5, A7, A8, and A9
-https://github.com/blacksphere/blackmagic/blob/master/src/target/adiv5.c#L202
+https://github.com/blackmagic-debug/blackmagic/blob/master/src/target/adiv5.c#L202
```

Comment:
`master` also needs changing to `main`. It might be worthwhile finding what these lines were supposed to refer to and grabbing a GitHub Permalink instead.

Reply:
I have updated master to main. I'll leave finding out what these lines were supposed to refer to as future work.

Reply:
Ok, that's fair enough - we'll look at trying to figure out what those should be linked to later then